src/main/java/org/openjdk/jextract/impl/ClassSourceBuilder.java

```diff
@@ -235,31 +240,32 @@ String indentString(int size) {
 
     private String primitiveLayoutString(Primitive primitiveType, long align) {
         return switch (primitiveType.kind()) {
-            case Bool -> STR."\{runtimeHelperName()}.C_BOOL";
-            case Char -> STR."\{runtimeHelperName()}.C_CHAR";
-            case Short -> alignIfNeeded(STR."\{runtimeHelperName()}.C_SHORT", 2, align);
-            case Int -> alignIfNeeded(STR."\{runtimeHelperName()}.C_INT", 4, align);
-            case Long -> alignIfNeeded(STR."\{runtimeHelperName()}.C_LONG", TypeImpl.IS_WINDOWS ? 4 : 8, align);
-            case LongLong -> alignIfNeeded(STR."\{runtimeHelperName()}.C_LONG_LONG", 8, align);
-            case Float -> alignIfNeeded(STR."\{runtimeHelperName()}.C_FLOAT", 4, align);
-            case Double -> alignIfNeeded(STR."\{runtimeHelperName()}.C_DOUBLE", 8, align);
+            case Bool -> String.format("%1$s.C_BOOL", runtimeHelperName());
```

Comment:
is this more readable than `runtimeHelperName() + ".C_BOOL"` ? I know this is all subjective, but we have a lot of "simple" concatenation which now is using `String::format`. Perhaps worth cleaning up in a separate PR.